.github/workflows/ci.yml

```diff
@@ -1,11 +1,12 @@
-name: CI
+name: Build And Test
 on:
   - push
   - pull_request
 jobs:
   build_and_test:
     name: many-rs build and tests
     runs-on: ubuntu-latest
+    permissions: write-all
```

Comment:
Did you enable this as well?

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#enabling-workflows-for-private-repository-forks

If so, I believe this to be a security issue.

Reply:
That's not needed anymore and wasn't even working anyway.